.github/workflows/end2end.yaml

```diff
@@ -445,6 +447,16 @@ jobs:
       - name: Debug wait
         uses: ./.github/actions/debug-wait
         timeout-minutes: 60
+      - name: Upload results
+        if: "!cancelled() && env.TRUNK_TOKEN"
+        uses: trunk-io/analytics-uploader@main
+        with:
+          junit-paths: /artifacts/data/artifacts/*.xml
+          org-slug: ${{ github.repository_owner }}
+          token: ${{ env.TRUNK_TOKEN }}
```

Comment:
don't we need to set `repo-head-branch` ?

Reply:
yes, as discussed with the guys, best to leave it unless we are in a situation where we see it is not computed properly